torch/csrc/distributed/rpc/python_functions.cpp

```diff
@@ -24,6 +24,7 @@ namespace rpc {
 namespace {
 
 py::object toPyObj(const Message& message) {
+  py::gil_scoped_release release;
```

Comment:
For which call this is?

Reply:
Yes, it was for PythonRpcHandler. I updated the PR to remove `toPyObj` and added a `toIValue`, as I think former is no longer needed.